README.md

```diff
@@ -61,14 +61,14 @@ The course includes links to different learning materials - video courses, tutor
 
 ### Progress Reports
 
-We would like to know the progress of students along the way. We kindly ask you to post about you progress (finished tasks) in students chat.
+We would like to know the progress of students along the way. We kindly ask you to post about your progress (finished tasks) in students chat.
 
  ### Chats
 
 For your comfort we have created several telegram-chats:
 
 1. [FE Students | Kottans](https://t.me/joinchat/DmX0JBHVkEhV1us2HdMmpA) - chat for students, where you can discuss anything related to the current course
-1. [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) - if you need a help of one of the mentors, or maybe from all of them, you can ask there
+1. [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) - if you need the help of one of the mentors, or maybe from all of them, you can ask there
```

Comment:
```suggestion
1. [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) - if you need help of one of the mentors, or maybe from all of them, you can ask there
```

Reply:
I'm not 100 percent sure, but it's probably better this way.

Reply:
Yeah, tricky one. Agree, it's better without a/the. Should I update it in my local copy and make a request again?

Reply:
@Andrew-Omelchenko you can, if you want. And then make a force push into this PR.

Or owner of repo can just press "Commit suggestion".